crates/astria-core/src/sequencerblock/v1alpha1/block.rs

```diff
@@ -196,7 +199,7 @@ impl SequencerBlockError {
         Self(SequencerBlockErrorKind::NoRollupTransactionsRoot)
     }
 
-    fn incorrect_rollup_transactions_root_length(len: usize) -> Self {
+    fn incorrect_rollup_tx_root_length(len: usize) -> Self {
```

Comment:
This should match the name of the error variant. Change back.

Reply:
This was to avoid a clippy warning of too many lines in `SequencerBlock::try_from_block_info_and_data`, as the lengthened function name pushes the formatter to break the `map_err` in which it is called into 3 lines instead of one. Would it be best for me to try to refactor in this case and avoid a clippy exception? Thanks for the detailed feedback :)

Reply:
Yeah, that seems to a hilarious way to appease the linter with the downside of breaking symmetry with all other methods. :-) The lint hitting that other function is probably correct. But I'm happy with you adding an `allow` directive on top and address it in a future PR.